observability/monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -270,15 +270,26 @@ include::modules/monitoring-creating-scrape-sample-alerts.adoc[leveloffset=+2]
 * See xref:../../observability/monitoring/troubleshooting-monitoring-issues.adoc#determining-why-prometheus-is-consuming-disk-space_troubleshooting-monitoring-issues[Determining why Prometheus is consuming a lot of disk space] for steps to query which metrics have the highest number of scrape samples.
 endif::openshift-dedicated,openshift-rosa[]
 
-//Configuring external alertmanagers
-include::modules/monitoring-configuring-external-alertmanagers.adoc[leveloffset=1]
+//Configuring external Alertmanager instances
+// The following module should only include core platform monitoring (CPM tags)
```

Comment:
Suggestion to move the adverb only as follows: The following module should include only core platform monitoring (CPM tags). Described in the IBM Style Guide here:  https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only

Reply:
Yes you are right thank you for the comment. I will keep it in mind in the future :+1: 
I will leave the current version for consistency (just because the lines will be removed in the nex steps of this restructuring project anyway and are here only for the reviewers to make it easier for them :grin: )